src/common/providers/https.ts

```diff
@@ -841,6 +842,14 @@ function wrapOnCallHandler<Req = any, Res = any>(
       }
 
       const data: Req = decode(req.body.data);
+      if (options.authPolicy) {
+        // Don't ask me why, but Google decided not to disambiguate between unauthenticated and unauthorized
```

Comment:
is this true? we have permission-denied and unauthenticated:

https://github.com/grpc/grpc/blob/master/doc/statuscodes.md

Reply:
I wrote this comment before I realized better. Will delete the comment.